java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java

```diff
@@ -28,6 +32,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(VirtualObservableReaderImpl.class);
 
+    /* The observers of this object */
+    private Set<ReaderObserver> observers;
```

Comment:
observers private attribute: the observers can't be managed though a 'Set', but only though a 'List'. Because the observers are unknown objects. A Set collection requires that the objects implements the methods 'equals': impossible if the objects are unknown.